### PR TITLE
Add VX300S Robot

### PR DESCRIPTION
**Pull request by cremebrule**

### CODE REVIEW

File: omnigibson/prims/rigid_prim.py [hunk 1]
```diff
@@ -201,7 +201,7 @@ def update_meshes(self):
                     # If we're not a valid volume, use bounding box approximation for the underlying collision approximation
                     if not is_volume:
                         log.warning(f"Got invalid (non-volume) collision mesh: {mesh.name}")
-                        mesh.set_collision_approximation("boundingCube")
+                        # mesh.set_collision_approximation("boundingCube")
```
Comment (ChengshuLi) [inline omnigibson/prims/rigid_prim.py:204]: is this no longer needed?

File: omnigibson/robots/vx300s.py [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+        # Not supported for this robot
+        raise NotImplementedError()
+
+    def _create_discrete_action_space(self):
+        # Fetch does not support discrete actions
+        raise ValueError("VX300S does not support discrete actions!")
+
+    def update_controller_mode(self):
+        super().update_controller_mode()
+        # overwrite joint params (e.g. damping, stiffness, max_effort) here
```
Comment (ChengshuLi) [inline omnigibson/robots/vx300s.py:138]: Maybe remove this function is there is no overwriting?

**Review by ChengshuLi — APPROVED** [review]
LGTM, left two minor comments, feel free to merge!

File: omnigibson/prims/rigid_prim.py (hunk 1 above)
Comment (cremebrule) [reply omnigibson/prims/rigid_prim.py:204]: Yes, per discussion with @cgokmen . I think our conclusion was that we should never do this anymore, or rather that (a) all of our assets should never have this problem, and (b) any edge cases we encounter we should explicitly address

File: omnigibson/robots/vx300s.py (hunk 2 above)
Comment (cremebrule) [reply omnigibson/robots/vx300s.py]: done!

**Pull request merged**